### ISSUE TITLE
Empty list in feature importance 

### ISSUE THREAD

**Issue by samshipengs** [catboost/catboost#831]
**Problem:** 
I'm trying to obtain feature importance from a trained multi-class classifier 
`clf.get_feature_importance(data=cat.Pool(data=x_trn, label=y_trn), type='FeatureImportance')`
but I got an empty list and a message shown:

> Dataset is provided, but PredictionValuesChange feature importance don't use it, since non-empty LeafWeights in model.

the training data size and validation size is:

>  train len=688,743 | val len=137,762

**Catboost version**: 0.14.2
**Operating System**: Ubuntu 18.04 LTS
**CPU**: Intel(R) Core(TM) i7-6700K CPU
**GPU:** GeForce GTX 1080

**Comment by annaveronika** [comment]
Thank you for the report, we'll look into this!

**Comment by annaveronika** [comment]
Could you please provide the full code that you are running? I tried to reproduce it, but the result that I see is not empty.

**Comment by samshipengs** [comment]
thanks for the quick reply, here is a bit more details
```
clf = cat.CatBoostClassifier(**params)
clf.fit(x_trn, y_trn,
         eval_set=(x_val, y_val),
         early_stopping_rounds=100,
         verbose=100,
         plot=False)
trn_imp = clf.get_feature_importance(data=cat.Pool(data=x_trn, label=y_trn),
                                        prettified=True,
                                        type='FeatureImportance')
```

where `params` is:
```
params = {'loss_function': 'MultiClass',
              'custom_metric': ['MultiClass', 'Accuracy'],
              'eval_metric': 'MultiClass',
              'iterations': 3000,
              'learning_rate': 0.02,
              'depth': 8,
              'task_type':'GPU'}
```

**Comment by annaveronika** [comment]
you don't have to specify MultiClass in custom_metric, objective function  and eval_metric are always calculated.

**Comment by annaveronika** [comment]
Ok, thanks for the example, we'll check.

**Comment by ek-ak** [comment]
Hi! Could you please provide minimal representative example with some pool? I failed to reproduce empty fstr result with your code.

Warning
> Dataset is provided, but PredictionValuesChange feature importance don't use it, since non-empty LeafWeights in model.

means, that _data_ parameter is not required for _get_feature_importance_ function for Multiclass.

**Comment by avivnahon** [comment]
I have the same warning when training a model. I run:
`train_pool <- catboost::catboost.load_pool(data = x, label = y, cat_features = category_indices)`
  `params <- list(...)`
  `catboost::catboost.train(learn_pool = train_pool, params = params)`

After training is over, I get the following warning:
`Dataset is provided, but PredictionValuesChange feature importance don't use it, since non-empty LeafWeights in model.`

What can be the reason? 
Thanks

**Comment by samshipengs** [comment]
@ek-ak Thanks for explaining the warning, hmm after I tried with more data in my training, the feature importance works correctly then (non-empty).
